db/migrate/20170711025530_add_published_to_collections.rb

```diff
@@ -0,0 +1,5 @@
+class AddPublishedToCollections < ActiveRecord::Migration
+  def change
+    add_column :collections, :is_published, :boolean, null: false, default: false
```

Comment:
just an idea: some people like to store this kind of thing not as a boolean, but as a timestamp, ie, `#published_at`. you can then add a method to the class like:

```ruby
def published?
  !!published_at
end
```

maybe consider that if you feel the published date/time might be a bit of useful info for them